### ISSUE TITLE
Add new columns for types of travel

### ISSUE THREAD

**Issue by m-kruth** [2kruman/COVID19-NZ-known-cases#28]
There appears to be new case types for people who:

- Travelled only domestically
- Cruise travel
- Did not travel

Columns to capture each of these (and a new one for international travel too) would be good. Each column can be a boolean `TRUE` or `FALSE`, so that we can easily capture types of people who travelled in more than one form (e.g. _international and domestic travel_ or _cruise and domestic_).

**Comment by RapidCompiler** [comment]
I think we should add six columns for addressing both issue #29 and issue #28 together. The main data set should include six columns namely Community Trans., Travel, International, Domestic, Flight, Cruise Ship. What do you think?

**Comment by m-kruth** [comment]
Firstly my apologies for not being active over the last day. I've had some family commitments that I've needed to tend to.
I agree with the introduction of the following columns: `Community Transmission`, `International`, `Domestic` and `Cruise`.

I believe that a `Travel` column introduces unnecessary verbosity, if all of the listed above are FALSE than travel can also be deduced to be FALSE.

My opinion is the same for a `Flight` column. If international and domestic are FALSE, then flight will also be FALSE. 

While I can see the benefit of these columns, I think that we should leave it to users to extrapolate these. Another way to think of it is: removal of these columns results in no loss of information, they can still be extrapolated by looking at the other columns suggested above.

I'm open to discussion on this.

**Comment by RapidCompiler** [comment]
Yes, that is true. I am currently working on this. I will add the columns that you mentioned and update the data set accordingly.

**Comment by RapidCompiler** [comment]
It might take a while tho... Is that fine with you?

**Comment by m-kruth** [comment]
Absolutely take your time, quality over speed.
